jupyter-js-widgets/src/utils.js

```diff
@@ -66,11 +66,18 @@ function loadClass(class_name, module_name, registry) {
 
             // If the module is jupyter-js-widgets, we can just self import.
             var modulePromise;
-            if (module_name === 'jupyter-js-widgets') {
-                modulePromise = Promise.resolve(require('../'));
+            var requireJSDefined = Boolean(typeof window !== 'undefined' && window.requirejs);
```

Comment:
In strict mode, you can always undefined was not defined right? Then you should be able to compare with undefined?